Assets/ZSSRichTextEditor.js

```diff
@@ -888,15 +907,29 @@ ZSSEditor.removeImage = function(imageNodeIdentifier) {
     if (imageNode.length != 0){
         imageNode.remove();
     }
-
     // if image is inside options container we need to remove the container
     var imageContainerNode = this.getImageContainerNodeWithIdentifier(imageNodeIdentifier);
     if (imageContainerNode.length != 0){
+        //reset id before removal to avoid detection of user removal
+        imageContainerNode.attr("id","");
         imageContainerNode.remove();
     }
 };
 
 /**
+ *  @brief      Callbacks to native that the media container was deleted by the user
+ *
+ *  @param      mediaNodeIdentifier     The unique media ID
+ */
+ZSSEditor.sendMediaRemovedCallback = function( mediaNodeIdentifier ) {
```

Comment:
Remove the extra spacing between parenthesis and parameters.

Do the same for the calls inside this method.